### DISCUSSION TITLE
Feature Request: seek shortcuts

### DISCUSSION THREAD

**Discussion by sandersantema** [fooyin/fooyin#397] [Ideas]
I'd like to be able to seek in a track using shortcuts, i.e. skip forward by some time of my choosing.

Ideally I think would be the option to configure seeking:
- by a set amount of time i.e. 30 sec or,
- by a percentage of the total track time.

**Comment by z411** [comment]
I was also surprised to see I can't set shortcuts for seeking. Am I missing something?
